src/components/FileList/FileElement.js

```diff
@@ -62,15 +36,19 @@ const FileElement = ({
   const fileNameClass = classes(
     {
       'FileStatus--loading': loading,
-      FileStatus: !loading,
+      'FileList--Element-name': !loading,
```

Comment:
Would `name` be the element in the BEM format?

Reply:
not sure, we really need to nail down our naming conventions and document them in the kf-uikit